_source/logzio_collections/_metrics-sources/postgresql.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Ship PostgreSQL metrics
+logo:
+  logofile: 
+  orientation: vertical
+data-source: PostgreSQL
+contributors:
+  - shalper
+  - yotamloe
+shipping-tags:
+  - platform-service
+---
+
+You can ship PostgreSQL metrics using Metricbeat.
+
+#### Metricbeat setup
+
+**Before you begin, you'll need**:
+[Metricbeat 7.1](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-installation.html) or higher
+
+<div class="tasklist">
+
+##### Download the Logz.io public certificate
+
+For HTTPS shipping, download the Logz.io public certificate to your certificate authority folder.
+
+```shell
+sudo curl https://raw.githubusercontent.com/logzio/public-certificates/master/TrustExternalCARoot_and_USERTrustRSAAAACA.crt --create-dirs -o /etc/pki/tls/certs/COMODORSADomainValidationSecureServerCA.crt
+```
+
+##### Add Logz.io configuration
+
+Replace the General configuration with Logz.io settings.
+
+{% include metric-shipping/replace-metrics-token.html %}
+
+```shell
+# ===== General =====
+fields:
+  logzio_codec: json
+  token: <<SHIPPING-TOKEN>>
+fields_under_root: true
+```
+
```

Comment:
@yotamloe Which file is this done in? Is this the general metricbeat configuration yaml file?